src/WeekComponent/WeekComponent.css

```diff
@@ -6,76 +6,99 @@
 .description-week-day {
   height: 26px;
   align-self: center;
-  padding: 10px;
+  padding: 15px 178px 24px 0px;
   letter-spacing: 1.19px;
-  font-family: "San Francisco";
+  font-family: 'arial';
+  font-weight: bold;
 }
 
 .pause-start-button {
-  width: 159px;
-  height: 52px;
+  width: 336px;
+  height: 60px;
   border-radius: 25px;
   border: solid 6px rgb(233, 225, 247);
   background-color:  rgb(156, 115, 219);
   cursor: pointer;
   color: white;
   align-self: center;
   margin-top: 10px;
+  font-family: sans-serif;
+  font-size: 18px;
+  font-weight: bold;
 }
 
-.reset-button {
-  width: 159px;
+.reset-switch-button {
+  width: 336px;
   height: 52px;
```

Comment:
Not related to button, but check the progress bar and fix the margins, paddings, heights, to make them proportional.

Reply:
Set the exercise data container to position: fixed and place it at the bottom of the screen.